libs/sdk-core/src/breez_services.rs

```diff
@@ -392,6 +394,21 @@ impl BreezServices {
         Ok(())
     }
 
+    pub async fn prepare_withdraw(
```

Comment:
In order to minimize backward incompatibility when changing BreezServices interface methods it is better to create a structure that consists of all the input parameters. In this case: `PrepareWithdrawRequest`

Reply:
Interesting, I'll do it, thanks Roei 🙏